packs/travisci/actions/list_repos.py

```diff
@@ -0,0 +1,19 @@
+from lib.action import TravisCI
+import requests
```

Comment:
I didn't see this lib in the `requirements.txt`. I know we ship this by default (https://github.com/StackStorm/st2/blob/master/requirements.txt#L17), but it's good to be explicit so the library is setup in the pack virtualenv in the event we decide to change libraries.